numba/cuda/tests/cudapy/test_ipc.py

```diff
@@ -180,7 +180,7 @@ def test_ipc_array(self):
         self.check_ipc_array(slice(3, 8))
         self.check_ipc_array(slice(None, 8))
 
-@linux_only
+@unittest.skipUnless(not_linux, 'Only on OS other than Linux')
```

Comment:
For double negative avoidance, would `@unittest.skipIf(linux)` be ok (where `linux` is defined like above, but without the `not`!?

Reply:
Yes, I got confused around this once already!

Reply:
Fix pushed, waiting for CI again.